### DISCUSSION TITLE
The path is not recognized correctly

### DISCUSSION THREAD

**Discussion by keria-Davis** [JanDeDobbeleer/oh-my-posh#2271] [Troubleshoot]
I used the installation command `winget install oh-my-posh`,It helped me automatically configure the environment variables, and downloaded the theme, then I configured the initialization settings `oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\powerlevel10k_lean.omp.json" | Invoke-Expression`, but the terminal reported the error.

**Comment by keria-Davis** [comment]


**Reply by lewis-yeung** [reply]
What happens if you use UTF-8 encoding in PowerShell?

```powershell
[console]::InputEncoding = [console]::OutputEncoding = New-Object System.Text.UTF8Encoding
```

用户名是中文，如果当前用的是 GBK 编码，可能存在问题。

**Reply by lewis-yeung** [reply]
Indeed it works in this case, however it can cause annoying problems in some apps that do not support UTF-8 since it enables the system-wide UTF-8 encoding.

Windows 的编码确实是个很大的历史遗留问题。😅
